airflow/www/app.py

```diff
@@ -261,7 +261,9 @@ def index(self):
         session = Session()
         DM = models.DagModel
         qry = session.query(DM).filter(~DM.is_subdag, DM.is_active).all()
+        paused = session.query(DM).filter(~DM.is_subdag, DM.is_paused).all()
```

Comment:
I don't think there's a need for this query as you'll be able to use dag.is_paused